aws/ec2/README.md

```diff
@@ -11,18 +11,19 @@
 This example uses Terraform to launch an EC2 instance, deploy the [SecretHub Demo App](https://secrethub.io/docs/start/getting-started/#consume) on it and provision it with the required secrets using the [SecretHub AWS Identity Provider](https://secrethub.io/docs/reference/aws/). 
 
 ## Prerequisites
-1. [Terraform](https://www.terraform.io/downloads.html) installed
-2. [SecretHub](https://secrethub.io/docs/start/getting-started/#install) installed
+1. [Terraform](https://www.terraform.io/downloads.html) installed along with the [SecretHub Provider](https://secrethub.io/docs/guides/terraform/#install)
 3. Correctly configured [AWS Credentials](https://www.terraform.io/docs/providers/aws/index.html#authentication)
 
+See [variables.tf](./variables.tf) for the variables you need to set.
```

Comment:
Are these required to set beforehand? Aren't you prompted for them if they're missing?

Reply:
If they are required prerequisites, shall we include it in the list to make that clear?

Reply:
You'll get prompted for them if they're missing, but they're in the prereqs section because you still need to have some stuff available before you can start. But yes, I'll add it to the list as point 3.